tools/topology/sof/abi.m4

```diff
@@ -0,0 +1,13 @@
+
+dnl ABI_INFO(name, major, minor, patch)
+define(`ABI_INFO',
+`SectionData."SOF_ABI" {'
+`	bytes "$2,$3,$4"'
+`}'
+`SectionManifest.STR($1) {'
+`	data ['
+`		SOF_ABI'
+`	]'
+`}')
+
+ABI_INFO(sof_manifest, 0x03, 0x02, 0x00)
```

Comment:
These 0x03, 0x02 and 0x1 should be defined as SOF_ABI_MAJOR, SOF_ABI_MINOR and SOF_ABI_PATCH which should be read from uapi/abi.h to make sure there is no delta between these two files.